frontend/src/tests/pages/HomePage.test.js

```diff
@@ -10,184 +10,194 @@ import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+  test("expected CSS properties", () => {
+    axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.userOnly);
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <HomePage />
+        </MemoryRouter>
+      </QueryClientProvider>
+    );
+
+    const title = screen.getByTestId("homePage-title");
+    expect(title).toHaveAttribute("style", "font-size: 75px; border-radius: 7px; background-color: white; opacity: 0.9;");
+    const info = screen.getByTestId("info-section");
+    expect(info).toHaveAttribute("style", "padding: 20px; border-radius: 7px; margin-top: 20px;");
+    const img = screen.getByAltText('CS156 Project');
+    expect(img).toBeInTheDocument();
+    expect(img).toHaveAttribute('src', 'https://www.startyourlab.com/img/setup/github-organization.svg');
+    expect(img).toHaveStyle({
+      width: '100%',
+      borderRadius: '7px',
+      marginTop: '20px'
     });
-    
-    // test('renders greeting for non-logged-in users correctly', () => {
-    //     // Mock the `useCurrentUser` hook to return a logged-out state
-    //     jest.mock("main/utils/currentUser", () => ({
-    //         useCurrentUser: jest.fn(() => ({ data: { loggedIn: false } }))
-    //     }));
-    
-    //     render(
-    //         <QueryClientProvider client={queryClient}>
-    //             <MemoryRouter>
-    //                 <HomePage />
-    //             </MemoryRouter>
-    //         </QueryClientProvider>
-    //     );
-    
-    //     const greetingElement = screen.getByTestId("homePage-title");
-    //     expect(greetingElement.textContent).toContain("Good afternoon, cgaucho");
-    // });  
+  });
+
+  test('shows greeting for logged-in users with dynamic time of day', () => {
+    axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.userOnly); // Adjust this if needed
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <HomePage />
+        </MemoryRouter>
+      </QueryClientProvider>
+    );
+
+    const greetingElement = screen.getByTestId("homePage-title");
+    expect(
+      greetingElement.textContent
+    ).toMatch(/Good (morning|afternoon|evening), cgaucho/);
+  });
+
+  // test('renders greeting for non-logged-in users correctly', () => {
+  //     // Mock the `useCurrentUser` hook to return a logged-out state
+  //     jest.mock("main/utils/currentUser", () => ({
+  //         useCurrentUser: jest.fn(() => ({ data: { loggedIn: false } }))
+  //     }));
+
+  //     render(
+  //         <QueryClientProvider client={queryClient}>
+  //             <MemoryRouter>
+  //                 <HomePage />
+  //             </MemoryRouter>
+  //         </QueryClientProvider>
+  //     );
+
+  //     const greetingElement = screen.getByTestId("homePage-title");
+  //     expect(greetingElement.textContent).toContain("Good afternoon, cgaucho");
+  // });  
```

Comment:
Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

Reply:
I see that this was in the original... it should not have been.  But since you've touched the file, you now have the responsibility to clean this up.